node/api/channel_ws_api.md

```diff
@@ -1233,9 +1234,30 @@ fall back to the latest mutually-signed state. Currently defined error codes are
 }
 ```
 
-The FSM will inform its client of each error, using a `conflict` report.
+#### Positive response
```

Comment:
This seems to be in the wrong place.

Reply:
One can send an error to a signing request and it will receive either a possitive or negative response: if one is indeed rejecting the right signing request - it would be this section here. If for some reason one tries aborting an update in an erronous manner (it is a deposit and one tries cancelling a withdrawal instead, one tries rejecting an update after they had already signed it or there is no ongoing update at all) - one would get the negative response instead